dhis-2/dhis-api/src/main/java/org/hisp/dhis/programstageworkinglist/ProgramStageQueryCriteria.java

```diff
@@ -102,5 +102,5 @@ public class ProgramStageQueryCriteria implements Serializable {
   private List<AttributeValueFilter> attributeValueFilters = Collections.emptyList();
 
   /** Property to filter events based on {@link org.hisp.dhis.program.Enrollment#followup property */
-  @JsonProperty private boolean followup;
+  @JsonProperty private boolean followUp;
```

Comment:
AFAIK before in the PR description you said that the property has to be called `followup` because of the class used for the DB model. Does this now need some kind of mapping somewhere?

Reply:
the front end converts the parameter to the API request, but I thought using `followup`  was consistent with the class name. So if we use `followUp` it is consistent with API although do not reflect the class property